spec/calendar.html

```diff
@@ -739,15 +758,14 @@ <h1>ISODateFromFields ( _fields_, _options_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal-isoyearmonthfromfields" aoid="ISOYearMonthFromFields">
-      <h1>ISOYearMonthFromFields ( _fields_, _options_ )</h1>
+      <h1>ISOYearMonthFromFields ( _fields_, _overflow_ )</h1>
       <p>
         The ISOYearMonthFromFields abstract operation implements the calendar-specific logic in the `Temporal.Calendar.prototype.yearMonthFromFields` method for the ISO 8601 calendar.
         It returns a Record with three fields ([[Year]], [[Month]], and [[ReferenceISODay]]). All are integers.
```

Comment:
Should we be more specific about sign here?

```suggestion
        It returns a Record with three fields ([[Year]], [[Month]], and [[ReferenceISODay]]). [[Month]], and [[ReferenceISODay]] are positive integers, while [[Year]] is a signed integer.
```